source/serverless-testing/README.rst

```diff
@@ -95,44 +92,55 @@ when using these variables to help prevent accidental leaks.
   authenticate to any Atlas Serverless instance created in the drivers testing
   Atlas group.
 
+.. _CLOUDP-117288: https://jira.mongodb.org/browse/CLOUDP-117288
+
 Existing Spec Tests
 ===================
 
-Tests defined in the following specifications MUST be included in a driver's
-Atlas Serverless testing suite, including prose tests:
+Unified spec tests from all specifications MUST be run against Atlas Serverless.
+Since schema version 1.4, unified tests can specify Atlas Serverless
+compatibility in their ``runOnRequirements``.
+
+Any prose and legacy spec tests defined in the following specifications MUST be
+included in a driver's Atlas Serverless testing suite:
 
-- CRUD, including the v1 and unified tests
+- CRUD
+- Load Balancer
 - Retryable Reads
 - Retryable Writes
-- Versioned/Stable API
-- Driver Sessions
+- Sessions
 - Transactions (excluding convenient API)
 
-    - Note: the killAllSessions command is not supported on Serverless, so the
-      transactions tests may hang if an individual test leaves a transaction open
-      when it finishes (`CLOUDP-84298 <https://jira.mongodb.org/browse/CLOUDP-84298>`_)
-- Load Balancer unified tests
+  - Note: the killAllSessions command is not supported on Serverless, so the
+    transactions tests may hang if an individual test leaves a transaction open
+    when it finishes (`CLOUDP-84298`_).
+- Versioned/Stable API
 
-.. _create-instance.sh: https://github.com/mongodb-labs/drivers-evergreen-tools/blob/1ca6209825b6ed07ce90e24cda659143443709c8/.evergreen/serverless/create-instance.sh
+.. _CLOUDP-84298: https://jira.mongodb.org/browse/CLOUDP-84298
 
-Note that the formats for the JSON/YAML tests of these specifications were
+Note that the legacy JSON/YAML test formats for these specifications were
 updated to include a new ``runOnRequirement`` specifically for Atlas Serverless
-testing, so make sure to update the affected test runners to support this
-requirement and then sync the tests. To ensure these requirements are enforced
-properly, the runner MUST be informed that it is running against an Atlas
-Serverless instance through some indicator (e.g. an environment variable).
+testing. To ensure these requirements are enforced properly, the runner MUST be
+informed that it is running against an Atlas Serverless instance through some
+indicator (e.g. an environment variable).
```

Comment:
I removed the following, as it only seemed relevant to the original implementation of serverless testing:

> so make sure to update the affected test runners to support this requirement and then sync the tests